docs/user_manual.md

```diff
@@ -139,6 +140,33 @@ A current best practice is to save all your job creation parameters in
 version-controlled files in your project directory. This allows you to tie
 your helios job params to changes in your application code.
 
+### Health Checks
+
+When a job is started, Helios can optionally run a health check before registering your service with
+nameless. This prevents the service from receiving traffic before it is ready. After the container
```

Comment:
replace nameless with service discovery or service registrar. "before registering your service with whatever service registration plugin you provided."